JumpscaleCore/servers/threebot/base_actors/gdrive.py

```diff
@@ -1,4 +1,6 @@
 from Jumpscale import j
+from googleapiclient.errors import HttpError as GoogleApiHTTPError
```

Comment:
is that part of jumpscale installation?

Reply:
do not know for sure when it's installed, it's used originally by gdrive client.

Reply:
please update the actor code with try/except and install the python api until we add it as a dependency in the installation and have an image ready with it